src/components/edit-user.html

```diff
@@ -1,95 +1,142 @@
+      }
 
-            button:hover {
-                background-color: rgb(196, 226, 245);
-            }
-        </style>
-
-        <h1 class="div-center">Edit User</h1>
-        <div>
-            <div class="container">
-                <div class="form-border">
-                    <form>
-                        <label for="fname">First Name</label>
-                        <input type="text" id="fname" name="firstname" placeholder="Your name..">
-
-                        <label for="lname">Last Name</label>
-                        <input type="text" id="lname" name="lastname" placeholder="Your last name..">
-
-                        <label for="phone">Phone</label>
-                        <input type="text" id="phone" name="phone" placeholder="Your phone number..">
-
-                        <label for="email">Email</label>
-                        <input type="text" id="email" name="email" placeholder="Your email..">
-                        <div>
-                            <button class="button">Edit</button>
-                        </div>
-                    </form>
-                </div>
-            </div>
+      .border {
+        border: #666 1px solid;
+        width: 20%;
+        margin: auto;
+        margin-top: 25px;
+        padding-bottom: 75px;
+      }
+
+      .button {
+        background-color: #87CEFA;
+        border: none;
+        color: white;
+        padding: 5px 12px;
+        text-align: center;
+        text-decoration: none;
+        display: block;
+        margin: auto;
+        font-size: 16px;
+        margin-top: 25px;
+        border-radius: 8px;
+      }
+
+      button:hover {
+        background-color: rgb(196, 226, 245);
+      }
+
+      .container {
+        padding: 10px 16px;
+        width: 80%;
+      }
+
+      .container::after,
+      .row::after {
+        content: "";
+        clear: both;
+        display: table;
+      }
+
+      .card {
+        box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+        padding-top: initial;
+      }
+    </style>
+
+
+    <div class="card">
+      <div class="container">
+        <div class="form-border">
+          <h1 class="div-center">Edit User</h1>
+          <form>
+            <label for="fname">Enter User Id</label>
+            <input type="text" placeholder="Enter the user's id to delete or edit..." value="{{user.userId::input}}">
+
+            <label for="fname">Edit Name</label>
+            <input type="text" placeholder="Enter edit name..(key)" value="{{user.editName::input}}">
+
+            <label for="lname">Edit Value</label>
+            <input type="text" placeholder="Enter edit value..(value)" value="{{user.editValue::input}}">
+          </form>
         </div>
-    </template>
+        <button class="button" on-click="updateUser">Edit User</button>
+        <button class="button" on-click="deleteUser">Delete User</button>
+        <button class="button" on-click="deleteAllUsers">Delete All Users</button>
```

Comment:
I don't think this "Delete all user" button belongs here. Makes more sense in the users list or intern app components.